### PR TITLE
del

### PR DESCRIPTION
**Pull request by XingKaihang**

### CODE REVIEW

File: versioninfo.json [hunk 1]
```diff
@@ -1,14 +1,3 @@
-{
-	"FixedFileInfo":
-	{
```
Comment (XingKaihang) [inline versioninfo.json]: LGTM222

Comment (XingKaihang) [inline versioninfo.json]: M2

File: versioninfo.json [hunk 2]
```diff
@@ -1,14 +1,3 @@
...
-		"FileVersion": {
-			"Major": 2,
-			"Minor": 4,
```
Comment (XingKaihang) [inline versioninfo.json]: EN...

File: versioninfo.json [hunk 3]
```diff
@@ -1,14 +1,3 @@
...
-			"Patch": 0,
-			"Build": 0
-		}
```
Comment (XingKaihang) [inline versioninfo.json]: 批量提交评论

**Comment by XingKaihang** [comment]
aaa

File: versioninfo.json [hunk 4]
```diff
@@ -1,19 +1,8 @@
...
-	},
-	"StringFileInfo":
 	{
 		"FileDescription": "Git LFS",
 		"LegalCopyright": "GitHub, Inc. and Git LFS contributors",
```
Comment (XingKaihang) [inline versioninfo.json]: 1111

File: versioninfo.json [hunk 5]
```diff
@@ -1,19 +1,8 @@
...
 		"ProductName": "Git Large File Storage (LFS)",
 		"ProductVersion": "2.4.0"
```
Comment (XingKaihang) [inline versioninfo.json]: 2222

File: versioninfo.json [hunk 6]
```diff
@@ -1,19 +1,8 @@
-{
-	"FixedFileInfo":
-	{
```
Comment (chenyan19209219) [inline versioninfo.json]: asdfghjsdf

File: versioninfo.json [hunk 7]
```diff
@@ -1,19 +1,8 @@
...
-		"FileVersion": {
```
Comment (chenyan19209219) [inline versioninfo.json]: sdfadsfadf

File: versioninfo.json [hunk 8]
```diff
@@ -1,19 +1,8 @@
...
-			"Major": 2,
```
Comment (chenyan19209219) [inline versioninfo.json]: asdfasdafaf

File: versioninfo.json [hunk 9]
```diff
@@ -1,19 +1,8 @@
...
-			"Minor": 4,
-			"Patch": 0,
-			"Build": 0
```
Comment (chenyan19209219) [inline versioninfo.json]: sdfadfadsfa

**Pull request merged**